elasticsearch/src/main/java/org/hibernate/search/elasticsearch/client/impl/JestClient.java

```diff
@@ -119,9 +119,23 @@ public void start(Properties properties, BuildContext context) {
 						),
 						TimeUnit.SECONDS
 				)
-				.gson( gsonService.getGson() )
-				.build()
+				.gson( gsonService.getGson() );
+
+		String username = ConfigurationParseHelper.getString(
+				properties,
+				CLIENT_PROP_PREFIX + ElasticsearchEnvironment.SERVER_USERNAME,
+				null
 		);
+		if ( !StringHelper.isEmpty( username ) ) {
+			String password = ConfigurationParseHelper.getString(
```

Comment:
what if the username is non-empty, but the password is. Would that be valid or should we warn / ignore?